.github/workflows/auth-e2e-tests.yml

```diff
@@ -39,6 +39,9 @@ jobs:
 
           cat ${ENV_FILE_PATH}
 
+      - name: Build shared-utils
```

Comment:
Why do you have to do this?

Reply:
I may not have to. I tried getting turbo to manage it, through `dependsOn`, in the turbo.json file, but I couldn't get it to work as I expected it to work. I'm going to try again so we don't need this step in the workflow files.